flutter/lib/benchmark/resource_manager.dart

```diff
@@ -374,20 +374,24 @@ class ResourceManager {
   Future<List<BenchmarksConfiguration>>
       getAvailableBenchmarksConfigurations() async {
     final file = File('$applicationDirectory/$_configurationsFileName');
-    final benchmarksConfigurations = <BenchmarksConfiguration>[
-      defaultBenchmarksConfiguration
-    ];
-
-    if (await file.exists()) {
-      final content =
-          jsonDecode(await file.readAsString()) as Map<String, dynamic>;
-
-      for (final benchmarksConfigurationContent in content.entries) {
-        final benchmarksConfiguration = BenchmarksConfiguration(
-            benchmarksConfigurationContent.key,
-            benchmarksConfigurationContent.value as String);
-        benchmarksConfigurations.add(benchmarksConfiguration);
-      }
+    final benchmarksConfigurations = <BenchmarksConfiguration>[];
+
+    if (!await file.exists()) {
+      print('Create ' + file.path);
+      var config = <String, String>{
+        defaultBenchmarksConfiguration.name: defaultBenchmarksConfiguration.path
+      };
+      await file.writeAsString(jsonEncode(config));
```

Comment:
Function `getAvailableBenchmarksConfigurations` is only called when someone opens the list of available configurations.
Could you move this code to initialization stage?
I think `BenchmarkState::create` function would be convenient for this.

Reply:
Good catch. Thanks.